kani-driver/src/args.rs

```diff
@@ -114,6 +114,9 @@ pub struct KaniArgs {
     /// Kani will only compile the crate. No verification will be performed
     #[structopt(long, hidden_short_help(true))]
     pub only_codegen: bool,
+    /// Compile all features with kani.
```

Comment:
Can you please improve the doc? This is the cargo one:
> `--all-features`: Activates all features of all packages selected on the command-line.

Reply:
Addressed with 613160e1b55b08937351a19581ed56bbf6bb5874.